sphinx_automodapi/automodsumm.py

```diff
@@ -294,7 +294,8 @@ def automodsumm_to_autosummary_lines(fn, app):
         # Note: we use __name__ here instead of just writing the module name in
         #       case this extension is bundled into another package
         from . import automodapi
-        if automodapi.__name__ in app._extensions:
+
+        if automodapi.__name__ in getattr(app, '_extensions', 'extensions'):
```

Comment:
except that I agree with @cdeil, we should go for ``.extensions`` first rather than ``._extensions``